addons/medical/Language/ACE_Medical_localization.st

```diff
@@ -81,16 +88,5 @@ StringTable {
    Author "kexanone"
    LastChanged "apollo-jk"
   }
-  }
-  CustomStringTableItem "{5E101C5589E004EE}" {
-   Id "ACE_Medical-Item_Morphine_Description"
```

Comment:
Why does this entry get deleted?

Reply:
Not sure, wasn't able to get to localization issues tonight, will resolve later this week.

Reply:
Added the morphine description back in.